src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.scss

```diff
@@ -224,7 +224,7 @@
     .datatable-pager .pager {
       margin-right: 5px !important;
       & li:not(:first-child) {
-        /** Add space between buttons */
+        /* * Add space between buttons */
```

Comment:
Is this change intentional?

Reply:
No， it changed automatically after running the command 'npm run prettier' in my development platform.